default/scripting/policies/PROPAGANDA.focs.txt

```diff
@@ -31,7 +31,7 @@ Policy
                 Not EmpireHasAdoptedPolicy empire = LocalCandidate.Owner name = ThisPolicy
                 Number low = 1 condition = And [
                     OwnedBy empire = Source.Owner
-                    VisibleToEmpire empire = Target.Owner
+                    VisibleToEmpire empire = LocalCandidate.Owner
```

Comment:
This is maybe Source.Owner
If the empire with the policy does not see any planet of the LocalCandidate empire, then it can't not affect it. If LocalCadidate doesn't see any planet of Source.Owner is irrelevant (and play nice with stealth planets).

Reply:
> anyone listening in (=having visibility of any source object) would be affected by the propaganda...

That's the idea